src/ddbToEs/ddbToEsHelper.ts

```diff
@@ -42,57 +43,89 @@ export default class DdbToEsHelper {
         });
     }
 
-    async createIndexAndAliasIfNotExist(indexName: string) {
-        logger.debug('entering create index function');
+    async createIndexAndAliasIfNotExist(aliases: Set<string>) {
+        if (aliases.size === 0) {
+            return;
+        }
+
+        const listOfAliases = Array.from(aliases);
+        const { body: allFound } = await this.ElasticSearch.indices.existsAlias({
+            name: listOfAliases,
+            expand_wildcards: 'all',
+        });
+        if (allFound) {
+            // All needed aliases exist
+            return;
+        }
+
+        logger.debug('There are missing aliases');
+
+        const indicesToCreate: Set<string> = new Set();
+        listOfAliases.forEach((alias: string) => {
+            indicesToCreate.add(alias.substring(0, alias.length - 6)); // remove '-alias'
```

Comment:
I feel that there's a bit too much string manipulation to come up with the index name and aliases. Very soon we'll need to update this code to account for the multi-tenant aliases (that follow a different naming convention) so it'd be nice to DRY index/alias naming 

My recommendation would be to update this method to accept resourceTypes instead of aliases:
```
createIndexAndAliasIfNotExist(resourceTypes: Set<string>)
```
I think it's clearer to derive the index/alias name from the resource type than trying to get the index name by removing chars from the alias.

Also, I suggest adding a function:
```
const getAliasName = (resourceType: string): string
``` 
to be the single place where the alias naming convention is defined instead of having string concats across the codebase(i.e. `${lowercaseResourceType}-alias`)
https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/blob/79c77f878bfb5eb17cafe82a28c24c5b340ecdfa/src/ddbToEs/ddbToEsHelper.ts#L22



soon we'll need to update this code to handle the multi-tenant aliases as well